src/app/dataset/depublication/depublication.component.html

```diff
@@ -0,0 +1,113 @@
+<div class="depublication-ctrls">
+  <app-search
+    (onExecute)="setDataFilterParameter($event)"
+    [placeholderKey]="'depublicationFormPlaceholderSearchRecord'"
+  >
+  </app-search>
+
+  <ul class="filter">
+    <li *ngIf="isSaving">
+      <span class="svg-icon-spin showing"></span>
+    </li>
+    <li class="dropdown dropdown-options">
+      <a (click)="toggleMenuOptions()">{{ 'depublicationMenuTitle' | translate }}</a>
+      <div *ngIf="optionsOpen" class="dropdown-wrapper">
+        <ul class="dropdown-content">
+          <li>
+            <a (click)="openDialogFile()">{{ 'depublicationMenuFromFile' | translate }}</a>
+          </li>
+          <li>
+            <a (click)="openDialogInput()">{{ 'depublicationMenuFromInput' | translate }}</a>
+          </li>
+        </ul>
+      </div>
+    </li>
+  </ul>
+</div>
+<div class="modal-wrapper" *ngIf="dialogInputOpen || dialogFileOpen">
+  <div
+    class="modal"
+    [ngClass]="{
+      'dialog-file': dialogFileOpen,
+      'dialog-input': dialogInputOpen
+    }"
+  >
+    <div class="head">
+      {{ 'depublicationModalTitle' | translate }} <a class="btn-close" (click)="closeDialogs()"></a>
+    </div>
+    <ng-container *ngIf="dialogFileOpen">
+      <form [formGroup]="formFile" (ngSubmit)="onSubmitFormFile()">
+        <app-file-upload
+          formControlName="depublicationFile"
```

Comment:
My IDE complains that the declaration of `formControlName` is not necessary (not used? not allowed?)

Reply:
Providing `formControlName` exposes the required `onChange` hook needed for the handler.